docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,126 @@
+
+### Live mode
+
+Instantly see how the changes you introduce impact the test.
+
+### Debug mode
+
+Pause your tests to examine the app and troubleshoot errors.
+
+## Community-driven
+
+### Free and Open Source
+
+TestCafe is distributed for free under the MIT license.
+
+### Actively maintained
+
+The TestCafe team is quick to respond to GitHub issues and StackOverflow questions. Community suggestions shape the future of the product.
+
+### In-house support
+
+All TestCafe Studio Pro customers (subscribers and trial users alike) receive prompt, proactive support directly from the package maintainers.
+
+
+## How Does TestCafe Work?
+
+The automation engine behind TestCafe takes over browsers and the web pages they display to simulate user activity.
+
+### The Server-Client Architecture
+
+The server-client architecture of TestCafe allows it to execute system-level and in-browser code.
+
+* TestCafe uses [high-level system APIs](https://github.com/DevExpress/testcafe-browser-tools) to launch and manage browsers. This is necessary to achieve full control over test execution.
+* TestCafe tests are full-fledged Node.js scripts. They can launch services and applications, read and write system files, make use of your favourite libraries.
+* At the same time, TestCafe is capable of performing in-browser actions with client-side automation scripts. This is how TestCafe handles asynchronous events, simulates user activity, and executes custom (user-defined) JavaScript.
+
+Our experience shows that this hybrid approach gives users the best of both worlds. Early versions of TestCafe ran entirely in the browser. This caused a number of issues. The tests’ capabilities were limited, the application often interfered with page execution, and the sub-optimal logic separation caused browser crashes. Since then, we decoupled the engine, exposed the underlying Node.js API, and implemented a test runner that executes test scripts server-side.
+
+### Page Proxying: Bridging the Server-Client Gap
+
+You may notice that when you run TestCafe, the URL in the browser’s address bar does not match that of your website. This happens because TestCafe runs an under-the-hood reverse proxy.
+
+The [testcafe-hammerhead](https://github.com/DevExpress/testcafe-hammerhead) proxy intercepts browser requests and adds automation scripts to the requested resources. It then modifies the URLs contained within the resource, so that they point to the proxy. This way, neither the client-side code, nor the resources it communicates with, can tell that the page has been modified. To conceal automation scripts from the rest of the page code, TestCafe mocks the browser API.
+
+The proxying mechanism ensures that the page appears to be hosted at the original URL even to the test code. This is why you can use your website’s actual URL in tests, and pay no mind to the browser’s address bar.
```

Comment:
```suggestion
The proxying mechanism ensures that the page appears to be hosted at the original URL even to the test code. This is why you can use your website’s actual URL in tests, and pay no attention to the browser’s address bar.
```

Reply:
Tbh I'd simply remove the second sentence. It's redundant.
```suggestion
The proxying mechanism ensures that the page appears to be hosted at the original URL even to the test code.
```

Reply:
ok